printer_mods/dury10/Spartacus/README.MD

```diff
@@ -0,0 +1,115 @@
+
+
+
+## Spartacus embodies relentless printing power in a compact design. Featuring dual 4010 blowers for optimal cooling and a T-Volcano heat block with CHT Volcano nozzle, Spartacus delivers unrivaled flow rates for lightning-fast, high-quality prints. Its agile build allows for swift movements, maximizing print area and minimizing downtime. Dominate your printing projects with Spartacus: where strength meets precision.
+
+
+
+
+## Key Features:
+
+- Powerful direct drive extruder for rapid printing (Flow tested up to 55mm³/s)
+- Utilizes the T-Volcano hotend for increased flow and faster prints
+- Enhanced cooling with 3010 and dual 4010 GDS Time 12000 rpm fans
+- Lightweight and compact design inspired by Voron Design (Mini Stealth Burner)
+- Cost-efficient, with a total build cost under $75
+- Versatile design allows for customization and adaptation to different setups
+- 2 Neopixel LED lights for improved visibility and real-time print status updates
+
+## Bill of Materials (BOM):
+
+### Fans:
+
+- 1 x 3010 GDS Time 12000 rpm Fan
+- 2 x 4010 GDS Time Blower Fan (12000 rpm dual ball bearing recommended for optimal cooling)
+- 2 x Neopixel LED lights (Addressable LEDs)
+
+### Hotend Components:
+
+- 1 x V6 Heat Sink
+- 1 x BiMetal Heat Break
+- 1 x T-Volcano Heat Block 
+- 1 x Volcano Cht Nozzle
+- 1 x 65W Heater Cartridge (or higher)
+- 1 x Thermistor Cartridge for V6 Volcano Heater Block 3x15 (PT100 style)
+- 1 x Nema 14 Pancake Stepper Motor
+- 1 x BMG Dual Driver Gear Kit
+- 1 x 30mm PTFE Tube 1.9mm ID 4mm OD
+- 2 x Round PCB LED (WRGB, RGB, RGBW, WWRGB, etc.)
+
+### Optional Add-ons:
+
+- Piggyback/Breakout Board ($9) or Can Board ($20) for additional functionality
+- End caps for cable management based on your setup and chosen board
+
+### Nozzle Options (Optional, depending on preference):
+
+- T Volcano or Crazy Tvolcano Heat Block for easier nozzle changes and protect the heatbrake when printing at speeds.
+- Ceramic Heater or Revo High Flow for enhanced performance (cost considerations apply)
+
+## Hardware:
+
+### Heatset Inserts:
+
+- 1 x M3 x 5.7 x 4.5/4.6 Heatset Insert
+- 8 x M3 x 3 x 4.5 Heatset Insert
+
+### M3 Screws:
+
+- 3 x M3 x 6
+- 5 x M3 x 10
+- 3 x M3 x 25
+- 1 x M3 Hex Nut
+
+
+#### This versatile and feature-packed 3D printer tool head provides an affordable solution for high-quality, high-speed printing, enhanced by Neopixel LED lights for improved visibility and dynamic print status updates.
+
+#### Assembly of Spartacus requires minimal tools, including an 1.5mm drill bit, a drill, an angle grinder or hacksaw, and 1.5, 2 and 2.5mm Allen keys. With these tools, users can effortlessly assemble Spartacus and unleash its full potential.
```

Comment:
hacksaw and angle grinder aren't common tools for printer modding, when you add the assembly instructions, please be sure to explain their usage.

Reply:
I will do my best to make a detailed install with photos. for the V6 that I have suggested. I have also added Rapido UHF that doesn't need cutting or drilling. 
Thanks for your time to review. I was already uploading the changes and after that I saw your suggestions, thanks